modules/rhel-compute-overview.adoc

```diff
@@ -8,18 +8,18 @@
 [id="rhel-compute-overview_{context}"]
 = About adding RHEL compute nodes to a cluster
 
-In {product-title} {product-version}, you have the option of using Red Hat Enterprise Linux (RHEL) machines as compute machines, which are also known as worker machines, in your cluster if you use a user-provisioned infrastructure installation. You must use {op-system-first} machines for the control plane, or master, machines in your cluster.
+In {product-title} {product-version}, you have the option of using {op-system-base-full} machines as compute machines in your cluster if you use a user-provisioned infrastructure installation on the `x86_64` architecture. You must use {op-system-first} machines for the control plane machines in your cluster.
 
-As with all installations that use user-provisioned infrastructure, if you choose to use RHEL compute machines in your cluster, you take responsibility for all operating system life cycle management and maintenance, including performing system updates, applying patches, and completing all other required tasks.
+As with all installations that use user-provisioned infrastructure, if you choose to use {op-system-base} compute machines in your cluster, you take responsibility for all operating system life cycle management and maintenance, including performing system updates, applying patches, and completing all other required tasks.
 
 [IMPORTANT]
 ====
-Because removing {product-title} from a machine in the cluster requires destroying the operating system, you must use dedicated hardware for any RHEL machines that you add to the cluster.
+Because removing {product-title} from a machine in the cluster requires destroying the operating system, you must use dedicated hardware for any {op-system-base} machines that you add to the cluster.
 ====
 
 [IMPORTANT]
 ====
-Swap memory is disabled on all RHEL machines that you add to your {product-title} cluster. You cannot enable swap memory on these machines.
+Swap memory is disabled on all {op-system-base} machines that you add to your {product-title} cluster. You cannot enable swap memory on these machines.
```

Comment:
Since we try to avoid stacked admonitions, would you consider turning these two important admonitions into a bulleted list, as per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-notes)?

Reply:
@michaelryanpeter I changed to a bullet list. 
However, because I didn't fully understand the context of the first bullet, I continued reading and now I'm wondering. 
- Shouldn't the sentence go to the [system requirements](https://deploy-preview-45228--osdocs.netlify.app/openshift-enterprise/latest/machine_management/adding-rhel-compute.html#rhel-compute-requirements_adding-rhel-compute)? 
- Isn't that contradicting "use dedicated hardware"  versus "physical or virtual system"? To my understanding dedicated hardware is a  physical machine. 

Entry from system reqs: 
```
Each system must meet the following hardware requirements:
- Physical or virtual system, or an instance running on a public or private IaaS.
```